src/modules/ModuleGovernance/List/Proposal.vue

```diff
@@ -0,0 +1,87 @@
+<script setup lang="ts" name="ModuleGovernanceListProposal">
+import dayjs from 'dayjs'
+import { RouteName } from '@/types'
+import { Proposal } from '../types'
+
+const { t } = useI18n()
+const vBem = useBemClass()
+
+const props = defineProps<{
+  proposal: Proposal
+}>()
+const { proposal } = toRefs(props)
+
+function formatDate(timestamp: number) {
+  return dayjs(timestamp * 1000).format('DD.MM.YYYY')
+}
+
+const formattedStartDate = computed(() => {
+  return formatDate(proposal.value.start)
+})
+
+const formattedEndDate = computed(() => {
+  return formatDate(proposal.value.end)
+})
+</script>
+
+<template>
+  <router-link
+    v-bem
+    :to="{ name: RouteName.VotingProposal, params: { id: proposal.id } }"
+  >
+    <div v-bem="'item'">
+      <div v-bem="'item-label'">
+        {{ t('ModuleGovernanceListProposal.startDate') }}
+      </div>
+      <div v-bem="'item-value'">
+        {{ formattedStartDate }}
+      </div>
+    </div>
+    <div v-bem="'title'">
+      {{ proposal.title }}
+    </div>
+    <div v-bem="'item'">
+      <div v-bem="'item-label'">
+        {{ t('ModuleGovernanceListProposal.endDate') }}
+      </div>
+      <div v-bem="'item-value'">
+        {{ formattedEndDate }}
+      </div>
+    </div>
+    <ModuleGovernanceProposalStatus :status="proposal.status" />
+  </router-link>
```

Comment:
Mixing kebab `<router-link>` with pascal `<ModuleGovernanceProposalStatus>` within a single template is not a good practice. You should either kebab or pascal, not mix them. Unfortunately I can't find a detailed explanation for this in Vue Style Guide, but I am sure it is somewhere there.

Reply:
Oh, you are right. I'm not being considerate enough